tests/run.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
 
-source `dirname "${BASH_SOURCE[0]}"`/util.sh
-source `dirname "${BASH_SOURCE[0]}"`/init.sh $CLUSTER
+set -o errexit
+set -o nounset
+set -o pipefail
 
-if ! [ -x "$(command -v riff)" ]; then
-  # install riff if not present
-  go get github.com/projectriff/riff
-fi
+source `dirname "${BASH_SOURCE[0]}"`/../start.sh
+
+# install riff if not present
+go get github.com/projectriff/riff
```

Comment:
nit: could be in `.configure.sh`, or? every config will require riff.

Reply:
every run will need the riff cli, or something aliased as riff, but it's up to the runner to specify exactly which version of riff to use.